client/signup/steps/website-content/components.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from '@emotion/styled';
+import FormLabel from 'calypso/components/forms/form-label';
+
+export const HorizontalGrid = styled.div`
+	display: flex;
+	justify-content: space-between;
+	width: 627px;
+`;
+
+export const Label = styled( FormLabel )`
+	font-style: normal;
+	font-weight: 600;
+	font-size: 13px;
+	line-height: 20px;
+	text-align: center;
+	color: #2c3338;
```

Comment:
We can use this CSS var here.
```suggestion
	color: var( --studio-gray-80 );
```